convert2rhel/unit_tests/breadcrumbs_test.py

```diff
@@ -64,6 +64,8 @@ def test_finish_collection(pretend_os, success, monkeypatch):
 
     monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_migration_results", save_migration_results_mock)
     monkeypatch.setattr(breadcrumbs.breadcrumbs, "_save_rhsm_facts", save_rhsm_facts_mock)
+    # Set to true, pretend that user was informed about collecting data
+    breadcrumbs.breadcrumbs._inform_telemetry = True
```

Comment:
I think that maybe you should use monkeypatch here as well, because the way it is, you will override the that property for the instance of `breadcrumbs.breadcrumbs`. 

It is possible that if we forget about it, in another test we might get mixed results because of this. Will try this locally to see if that's really the case.

Reply:
Check out this results: 

```python
# Assuming at this point that _inform_telemetry is equal to True

def test_something():
    print("Value for _inform_telemetry", breadcrumbs.breadcrumbs._inform_telemetry)

def test_modify_value():
    breadcrumbs.breadcrumbs._inform_telemetry = False

def test_something_2():
    print("Value for _inform_telemetry", breadcrumbs.breadcrumbs._inform_telemetry)
```

Results

```
convert2rhel/unit_tests/breadcrumbs_test.py ..Value for _inform_telemetry True
..Value for _inform_telemetry False
.
```

You can see that this line of code can actually pollute other tests down the road.